dev/Dockerfile.base

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/app-sre/ubi8-ubi:8.2
+FROM quay.io/app-sre/ubi8-ubi:8.4
```

Comment:
Why is the image from `app-sre` mirror is used?

Reply:
I incorrectly believed that the image from `registry.access.redhat.com` required either subscription or a vpn connection.  Further testing proved otherwise.  Thanks for catching this, updating.